docs/RELEASE_NOTES.md

```diff
@@ -150,379 +146,299 @@ Use Cocos Console to create a new game:
 cocos new -l cpp|js|lua MyNewGame
 ```
 
-# v3.8
+# v3.9
 
-## Highlights and API changes of v3.8
+## Highlights features, improvements and API updates of v3.9
 
-As promised, Cocos2d-x v3.8 have greatly improved the stability and API friendliness. On one side, we continue to improve 3D capacity by providing 3D physics collider, skybox background brush and key frame event callback in Animate3D. On another side, we have refined several core modules, including UI system, AudioEngine, FileUtils, Bindings Generator, etc.
+We are happy to announce the release of Cocos2d-x v3.9. Following are the highlighted features, improvements and API updates in this version. 
 
-Here is some highlighted improvments and API changes:
+1. 3D Module: 
+    - Added 3D MotionStreak to support streak effect. 
+    - Refined Sprite3D to support material system. 
+2. 2D Module:
+    - Added frame callback function and animation callback function. 
+    - Added script component system. 
+    - Reconstruction of 2D physics with Component. 
+    - Improved EditBox implemention on iOS and Win32 platform.
+    - Removed dependence of libcurl on AssetsManager, AssetsManagerEx and Downloader (iOS & Android).
+    - Improved particle performance. 
+3. Others: 
+    - Supported Action inheritance, update function overwriting in JSB. 
+    - Improved ScrollView performance in Web engine. 
+    - Improved Scale9Sprite performance in Web engine. 
+    - Decoupled Sprite's setTexture and updateColor in Web engine.
+    - Added support for debugging and release on real devices with Xcode7 and iOS9.
 
-1. 3D Module
-    - Added 3D physics collider
-    - Supported setting camera background brushes with color/depth/skybox 
-    - Added key frame event Callback in Animate3D
-    - Added light map support in Terrain
-2. UI System
-    - Reimplemented and enhanced EditBox on Android
-    - Added ScrollViewBar for displaying a scroll bar at the side of ScrollView (JSB/Lua ready)
-    - Added RadioButton widget (JSB/Lua ready)
-    - Added HANYI FullType font support
-3. AudioEngine
-    - AudioEngine supported audio preloading
-    - Bound new AudioEngine in JSB
-4. FileUtils
-    - Added a set of file writing APIs: writeStringToFile, writeDataToFile, writeValueMapToFile, writeValueVectorToFile
-5. Others
-    - Improved Bindings Generator tool
-    - Merged JSB test project into cocos2d test project
-    - framework: Support generate prebuilt libs of engine with debug mode
-    - console: Supported new portrait projects from templates
+## The main features in detail of Cocos2d-x v3.9:
 
-## Download
+### 3D Module
 
-[Cocos2d-x v3.8](http://www.cocos2d-x.org/filedown/cocos2d-x-3.8.zip) including : C++, Lua & JS
+1. 3D MotionStreak
+	
+	In this version, 3D MotionStreak is added to support streak effect. Check the testcase of Sprite3Dtest to see how to use it.  
```

Comment:
Check the testcase: [Sprite3DTest](https://github.com/cocos2d/cocos2d-x/blob/v3/tests/cpp-tests/Classes/Sprite3DTest/Sprite3DTest.cpp#L2472) to see how to use it.